tools/cfn2ts/lib/codegen.ts

```diff
@@ -243,19 +254,11 @@ export default class CodeGenerator {
         attributes.push(refAttribute);
       }
     }
-    // set the TagType to help format tags later
-    const tagEnum = tagType(spec);
-    if (tagEnum !== `${TAG_TYPE}.NotTaggable`) {
-      this.code.line();
-      this.code.line('/**');
-      this.code.line(' * The `TagManager` handles setting, removing and formatting tags');
```

Comment:
Moved this into the emitProps call below. This comment is also removed which I think works. Should I bring this information back and update it because it is already out of date (which is my chief complaint with these type of comments).

Reply:
Totally find getting rid of the comment